common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -313,6 +352,43 @@ public boolean isActive() {
 
         @Override
         protected void renderBar(PoseStack poseStack, float renderY, float renderHeight, float progress) {
+            if (progress > 1) { // overflowing mana
+                float x1 = this.getRenderX();
+                float x2 = this.getRenderX() + this.getWidth();
+                int textureY1 = manaTexture.getTextureY1();
+                int textureY2 = manaTexture.getTextureY2();
+
+                int half = (textureY1 + textureY2) / 2 + (textureY2 - textureY1) % 2;
+                RenderUtils.drawProgressBarBackground(
```

Comment:
Why do we need to do this?

Reply:
This is just the background of the bar

Reply:
Yeah right we return after this. Got it now